config/core/test-configs.yaml

```
@@ -962,6 +962,12 @@ device_types:
       - blocklist: {defconfig: ['allmodconfig', 'multi_v5_defconfig']}
       - blocklist: {kernel: ['v3.']}
 
+  kontron-bl-imx8mm:
+    mach: freescale
+    class: arm64-dtb
+    dtb: 'freescale/imx8mm-kontron-n801x-s.dts'
```

```suggestion
    dtb: 'freescale/imx8mm-kontron-n801x-s.dtb'
```
Right now there aren't any jobs submitted to this new device. I can see this `freescale/imx8mm-kontron-n801x-s.dtb` file is listed in the artifacts so it should work with the suggested change above.
https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20220311.1/arm64/defconfig/gcc-10/artifacts.json

---

oh .. yes .. that has to be fixed. Thank you